### PR TITLE
Main v2

### PR DESCRIPTION
**Pull request by RussTIB**

### CODE REVIEW

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: hud-data/thughud.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ThugHud",
+  "author": "Russ",
+  "contributors": [
+    "N1te (Sex Hud creator)"
```
Comment (mastercoms) [inline hud-data/thughud.json:5]: Just have their name, not what they created

File: hud-data/thughud.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      "4/3",
+      "16/10",
+      "16/9"
+    ],
+    "position": [
+      "center"
+  ]
+  "repo": "https://github.com/RussTIB/thughud",
+  "hash": "90374efd3fa3c6c9717aec48f5dd4882d14ab903",
+  "prerelease": false,
```
Comment (mastercoms) [inline hud-data/thughud.json:50]: Don't need this

File: hud-data/thughud.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ThugHud",
+  "author": "Russ",
+  "contributors": [
+    "N1te (Sex Hud creator)"
+  ],
+  "social": {
+    "support": "https://steamcommunity.com/tradeoffer/new/?partner=413718379&token=3fJl5eiP",
+    "issues": "Add me on discord (russty._)",
```
Comment (mastercoms) [inline hud-data/thughud.json:9]: This needs to be a link, can use `https://discordapp.com/channels/@me/USERID`

File: hud-data/thughud.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    "steam_profile": "76561198373984107",
+    "steam_group": "https://steamcommunity.com/groups/TheEroticBarber",
+    "discord": "russty._",
```
[the remark this answers is not in the document]
Comment (mastercoms) [reply hud-data/thughud.json:12]: Your user ID is 592276117822636033

File: hud-data/thughud.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  "name": "ThugHud",
+  "author": "Russ",
+  "contributors": [
+    "N1te (Sex Hud creator)"
+  ],
+  "social": {
+    "support": "https://steamcommunity.com/tradeoffer/new/?partner=413718379&token=3fJl5eiP",
+    "issues": "Add me on discord (russty._)",
+    "steam_profile": "76561198373984107",
+    "steam_group": "https://steamcommunity.com/groups/TheEroticBarber",
```
[the remark this answers is not in the document]
Comment (RussTIB) [reply hud-data/thughud.json]: done

File: hud-data/thughud.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (RussTIB) [reply hud-data/thughud.json:12]: done

File: hud-data/thughud.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    "discord": "russty._",
+    "youtube": "@RusstyTF2",
+    "twitch": "Twitch.tv/russ_tf2",
```
[the remark this answers is not in the document]
Comment (RussTIB) [reply hud-data/thughud.json]: done

File: hud-data/thughud.json (hunk 1 above)
Comment (RussTIB) [reply hud-data/thughud.json]: done

File: hud-data/thughud.json (hunk 3 above)
Comment (RussTIB) [reply hud-data/thughud.json]: done

**Review by mastercoms — APPROVED** [review]

**Pull request merged**